yt_dlp/extractor/streamff.py

```diff
@@ -0,0 +1,46 @@
+# coding: utf-8
+from .common import InfoExtractor
+from ..utils import int_or_none, parse_iso8601
```

Comment:
just a suggestion (it's not needed per say tbh)
But i personally would have kept it same as all other extractors 
```suggestion
# coding: utf-8
from __future__ import unicode_literals

from .common import InfoExtractor
from ..utils import int_or_none, parse_iso8601
```

Reply:
No, if anything we should be removing the future imports from new extractors